src/nvim/ex_cmds2.c

```diff
@@ -392,12 +392,12 @@ int dbg_check_skipped(exarg_T *eap)
  * This is a grow-array of structs.
  */
 struct debuggy {
-  int dbg_nr;                   /* breakpoint number */
```

Comment:
This change and more before it change nothing except comment style, that's the kind of thing that can go in another PR. Maybe I'm being too picky. @bfredl what do you think, can we just leave it?